app/views.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template      
+from .schema import User
+from . import mongo
+
+main_blueprint = Blueprint('main', __name__)
```

Comment:
do not create blue print again , import it from route also you created this twice make it only in views.py '@main_blueprint.route('/users')'